chef/cookbooks/crowbar/recipes/crowbar-db-dump.rb

```diff
@@ -83,6 +83,10 @@
   # Stop remaining pacemaker resources
   bash "stop pacemaker resources" do
     code <<-EOF
+      crm stop postgresql
+      # HA: Deletes postgres data directory after stopping postgresql service
+      # but before stopping fs-postgresql.
+      rm -rf /var/lib/pgsql/data
```

Comment:
Why do you remove it within that block if you have later on the directory delete?